home/templates/home/modal.html

```diff
@@ -9,6 +9,23 @@ <h5 class="modal-title" id="formModalLabel">Join Penny University</h5>
             </div>
             <form method="post">
                 {% csrf_token %}
+                {% if invite_link %}
+                <div class="modal-body">
+                    <p class="font-weight-light">
+                        The Penny University community hangs out in Slack.
+                        Click the button below to go to our Slack sign-up page.
+                    </p>
+                    <p>Not sure yet?
+                            <a href="https://groups.google.com/forum/#!forum/penny-university">
+                                Check out what we've been up to on our forum.
+                            </a>
+                        </p>
```

Comment:
After looking at this in the context of the form, the "Not sure... Check out ... forum" text feels distracting from the goal of getting them into slack. Just delete it. (The forum will be gone soon anyway.)